### ISSUE TITLE
Overflow error on Ubuntu

### ISSUE THREAD

**Issue by mvorobyo** [larsmaaloee/deep-belief-nets-for-topic-modeling#7]
Hey I tried running your code but then got an overflow warning.
...
Processed 69 of 75 batches
Processed 70 of 75 batches
Processed 71 of 75 batches
Processed 72 of 75 batches
Processed 73 of 75 batches
Processed 74 of 75 batches
Processed 75 of 75 batches
Time  2.70161509514
Pre Training
Visible units: 2000 Hidden units: 500
[loggingutils.py       :38  ] - 2016-11-10 16:51:12,471 - WARNING - /home/michael/deep-belief-nets-for-topic-modeling/DBN/dbn.py:239: RuntimeWarning: overflow encountered in exp
  return 1. / (1 + exp(-x))

  File "./main.py", line 209, in <module>
    run_examples(options.exp_num)

  File "./main.py", line 195, in run_examples
    example1()

  File "./main.py", line 63, in example1
    deepbelief.run_pretraining(learning_rate=0.01, weight_cost=0.0002, momentum=0.9, gibbs_steps=1)

  File "/home/michael/deep-belief-nets-for-topic-modeling/DBN/dbn.py", line 56, in run_pretraining
    pretraining_process.start()

  File "/usr/lib/python2.7/multiprocessing/process.py", line 130, in start
    self._popen = Popen(self)

  File "/usr/lib/python2.7/multiprocessing/forking.py", line 126, in __init__
    code = process_obj._bootstrap()

  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()

  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)

  File "/home/michael/deep-belief-nets-for-topic-modeling/DBN/dbn.py", line 67, in run_pretraining_as_process
    r.rsm_learn(self.max_epochs)

  File "/home/michael/deep-belief-nets-for-topic-modeling/DBN/pretraining.py", line 102, in rsm_learn
    neg_vis, neg_hid_prob, D, p = self.__contrastive_divergence_rsm__(neg_vis, pos_hid_prob, D)

  File "/home/michael/deep-belief-nets-for-topic-modeling/DBN/pretraining.py", line 142, in __contrastive_divergence_rsm__
    neg_hid_prob = dbn.sigmoid(dot(neg_vis, self.weights) + outer(D, self.hidden_biases))

  File "/home/michael/deep-belief-nets-for-topic-modeling/DBN/dbn.py", line 239, in sigmoid
    return 1. / (1 + exp(-x))

  File "/home/michael/code/python/util/loggingutils.py", line 38, in my_showwarning
    logging.warning(warnings.formatwarning(message, category, filename, line) + '\n' + '\n'.join(traceback.format_stack()))

Epoch[ 1]: Perplexity = 584.66
Epoch[ 2]: Perplexity = 518.24
Epoch[ 3]: Perplexity = 475.66
Epoch[ 4]: Perplexity = 443.47
...
